crates/transaction-emitter-lib/src/transaction_generator/account_generator.rs

```diff
@@ -72,20 +63,19 @@ impl TransactionGenerator for AccountGenerator {
 
 #[derive(Debug)]
 pub struct AccountGeneratorCreator {
-    rng: StdRng,
     txn_factory: TransactionFactory,
 }
 
 impl AccountGeneratorCreator {
-    pub fn new(rng: StdRng, txn_factory: TransactionFactory) -> Self {
-        Self { rng, txn_factory }
+    pub fn new(txn_factory: TransactionFactory) -> Self {
+        Self { txn_factory }
     }
 }
 
 impl TransactionGeneratorCreator for AccountGeneratorCreator {
     fn create_transaction_generator(&self) -> Box<dyn TransactionGenerator> {
         Box::new(AccountGenerator::new(
-            self.rng.clone(),
+            StdRng::from_seed(OsRng.gen()),
```

Comment:
didn't quite understand this :P

Reply:
To avoid generating same address from different creators.

https://github.com/aptos-labs/aptos-core/blob/e17fa8823787cd81fba22aebaf30f71ff4466411/crates/transaction-emitter-lib/src/emitter/mod.rs#L309

Reply:
I see. Makes sense to me.